src/crypto.ts

```diff
@@ -47,6 +47,28 @@ function encodeNumber(num: number): Uint8Array {
   return encoded;
 }
 
+/**
+ * Converts the given bigint into a uint8 array.
+ *
+ * @param int - Bigint to encode.
+ * @returns - Bigint encoded as a byte array.
+ */
+export function encodeBigintAsUint64(int: bigint): Uint8Array {
+  // Assert the input is 64 bits.
+  const newint = BigInt.asUintN(64, int);
+  if (newint != int) {
+    throw new Error("Received int > 2^64-1");
```

Comment:
Add a current argument value to the error message, it might help debugging what value exceeded the limit.
```suggestion
    throw new Error(`Argument ${int} exceeds 2^64-1`);
```